src/handlers/payout/post.ts

```diff
@@ -119,7 +129,139 @@ export const incentivizeComments = async () => {
+  const payload = context.payload as Payload;
+  const issue = payload.issue;
+  if (!issue) {
+    logger.info(`Incomplete payload. issue: ${issue}`);
+    return;
+  }
+
+  if (issue.state_reason !== StateReason.COMPLETED) {
+    logger.info("incentivizePullRequestReviews: comment incentives skipped because the issue was not closed as completed");
+    // return;
+  }
+
+  if (paymentPermitMaxPrice == 0 || !paymentPermitMaxPrice) {
+    logger.info(`incentivizePullRequestReviews: skipping to generate permit2 url, reason: { paymentPermitMaxPrice: ${paymentPermitMaxPrice}}`);
+    // return;
+  }
+
+  const issueDetailed = bountyInfo(issue);
+  if (!issueDetailed.isBounty) {
+    logger.info(`incentivizePullRequestReviews: its not a bounty`);
+    // return;
+  }
+
+  const pulls = await getAllPullRequests(context, "closed");
+  if (pulls.length === 0) {
+    logger.debug(`incentivizePullRequestReviews: No pull requests found at this time`);
+    return;
+  }
+
+  let linkedPull;
+
+  for (const pull of pulls) {
+    const pullRequestLinked = await gitLinkedIssueParser({
+      owner: payload.repository.owner.login,
+      repo: payload.repository.name,
+      issue_number: pull.number,
+    });
+    const linkedIssueNumber = pullRequestLinked.substring(pullRequestLinked.lastIndexOf("/") + 1);
+    if (linkedIssueNumber === issue.number.toString()) {
+      linkedPull = pull;
+      break;
+    }
+  }
+
+  if (!linkedPull) {
+    logger.debug(`incentivizePullRequestReviews: No linked pull requests found`);
+    return;
+  }
+
+  const comments = await getAllIssueComments(issue.number);
+  const permitComments = comments.filter(
+    (content) => content.body.includes("Reviewer Rewards") && content.body.includes("https://pay.ubq.fi?claim=") && content.user.type == UserType.Bot
+  );
+  if (permitComments.length > 0) {
+    logger.info(`incentivizePullRequestReviews: skip to generate a permit url because it has been already posted`);
+    return;
+  }
+
+  const assignees = issue?.assignees ?? [];
+  const assignee = assignees.length > 0 ? assignees[0] : undefined;
+  if (!assignee) {
+    logger.info("incentivizePullRequestReviews: skipping payment permit generation because `assignee` is `undefined`.");
+    return;
+  }
+
+  const prReviews = await getAllPullRequestReviews(context, linkedPull.number);
+  logger.info(`Getting the PR reviews done. comments: ${JSON.stringify(prReviews)}`);
+  const prReviewsByUser: Record<string, string[]> = {};
+  for (const review of prReviews) {
+    const user = review.user;
+    if (!user) continue;
+    if (user.type == UserType.Bot || user.login == assignee) continue;
+    if (!review.body) {
+      logger.info(`incentivizePullRequestReviews: Skipping to parse the comment because body_html is undefined. comment: ${JSON.stringify(review)}`);
+      continue;
+    }
+    if (!prReviewsByUser[user.login]) {
+      prReviewsByUser[user.login] = [];
+    }
+    prReviewsByUser[user.login].push(review.body);
```

Comment:
`parseComments` needs the comment in HTML format but this is in Markdown format. Look at how I did it https://github.com/ubiquity/ubiquibot/blob/12bccaeea35636665f31e859fe011527801d8004/src/helpers/issue.ts#L212

Reply:
done.